virt-v2v/cold/entrypoint

```diff
@@ -53,26 +72,39 @@ for disk in /dev/block[0-9]* ; do
 	ln -s "$disk" "$DIR/$V2V_vmName-sd$(gen_name "$((num+1))")"
 done
 
-# Store password to file
-echo -n "$V2V_secretKey" > "$DIR/vmware.pw"
-args=("${args[@]}"
-    -ip "$DIR/vmware.pw"
-)
+if [ "$V2V_provider" == "vCenter" ] ; then
+  # Store password to file
+  echo -n "$V2V_secretKey" > "$DIR/vmware.pw"
+  args=("${args[@]}"
+      -ip "$DIR/vmware.pw"
+  )
 
-# Use VDDK if present
-if [ -d "/opt/vmware-vix-disklib-distrib" ]; then
-    args=("${args[@]}"
-        -it vddk
-        -io vddk-libdir=/opt/vmware-vix-disklib-distrib
-        -io "vddk-thumbprint=$V2V_thumbprint"
-    )
+  # Use VDDK if present
+  if [ -d "/opt/vmware-vix-disklib-distrib" ]; then
+      args=("${args[@]}"
+          -it vddk
+          -io vddk-libdir=/opt/vmware-vix-disklib-distrib
+          -io "vddk-thumbprint=$V2V_thumbprint"
+      )
+  fi
 fi
 
 echo "Starting virt-v2v"
 set -x
 ls -l "$DIR"
-exec virt-v2v -v -x \
-    -i libvirt \
-    -ic "$V2V_libvirtURL" \
-    "${args[@]}" \
-    -- "$V2V_vmName" |& /usr/local/bin/virt-v2v-monitor
+if [ "$V2V_provider" == "vCenter" ] ; then
+  exec virt-v2v -v -x \
+      -i libvirt \
+      -ic "$V2V_libvirtURL" \
+      "${args[@]}" \
+      -- "$V2V_vmName" |& /usr/local/bin/virt-v2v-monitor
+fi
+
+if [ "$V2V_provider" == "ova" ] ; then
+  exec virt-v2v -v -x \
+      -i ova "$V2V_diskPath" \
+      "${args[@]}"\
+      |& /usr/local/bin/virt-v2v-monitor
+fi
+
+qemu-img info /mnt/disks/disk0/disk.img
```

Comment:
added for debugging? if so, we should probably execute this only in debug mode

Reply:
and if so it should be within 2 loops (for block and file), like few lines above:
`for disk in /mnt/disks/disk[0-9]* ; do` ... `for disk in /dev/block[0-9]* ; do`
i think it might be nice to have even without being on debug.

Reply:
yes It's for debuging but I also think it's nice to have, will be easier to get the status in the end

Reply:
once we see it works, the output of this command will no longer be useful as the output from virt-v2v indicates whether the conversion completed successfully or not - the output of `qemu-img info` would then only indicate whether the data was written to the file we intended to write to.. it doesn't worth keeping it in production